tests/cli.rs

```diff
@@ -122,7 +123,8 @@ fn test_provide_get_loop(path: &Path, input: Input, output: Output) -> Result<()
             .arg("--addr")
             .arg(ADDR)
             .arg("--rpc-port")
-            .arg("0")
+            .arg("disabled")
+            .arg("--persistent=false")
```

Comment:
what about adding a test for the actual persistence being enabled?

Reply:
will do.